app.js

```diff
@@ -11,7 +11,82 @@ app.use(express.json());
 app.use(bodyParser.json());
 
 
+app.post('/register', (req, res) => {
+  const userId = randomUUID();
+  const newUser = {
+    id: userId,
+    email: "alex.gmail.com",
+    pass: "[pxnjyfgbcfnm"
+  }
+  res.set({'x-user-id': userId})
+  users.push(newUser);
+  res.status(201).send(users);
 
+});
+
+app.get('/products', (req, res) => {
+  res.send(products);
```

Comment:
Status code 200 is sent by default but anyway it's better to specify this code explicitly